### PR TITLE
Mass invite link

### PR DESCRIPTION
**Pull request by mattkrick**
TEST
- [ ] team invitation modal has a url that you can copy and send to your team
- [ ] there is no way on God's green earth that you can copy anything but the entirety of the url
- [ ] visiting that link as someone who's already on the team takes you to the team page
- [ ] visiting that link as someone who's a member of parabol but not on the team takes you to the team page
- [ ] visiting that link as someone who isn't logged in shows you a generic authentication page that defaults to the "Create Account" view
- [ ] Can switch views to "Sign In" and invite token is still in the URL
- [ ] Can switch views to "Forgot password" and invite token is still in the URL
- [ ] Can submit a forgotten password, go back, then go back to sign in & still the invite token is in the URL
- [ ] can login & get redirected to the team page
- [ ] can sign up and get redirected to the team page
- [ ] the authentication page background looks like a meeting always (it's prettier) & when creating an account, it redirects you to the meeting
- [ ] if you change the token to something wrong like "foo" in the url, it tells you the token is invalid
- [ ] if the token is expired (massInviteToken.ts, change LIFESPAN to eg `ms('1s')`, refresh modal to get new token link) it tells you the token is expired

### CODE REVIEW

**Comment by mattkrick** [comment]
@jordanh ready to test!

**Comment by jordanh** [comment]
## Test Cycle 1

### Tests
- [ ] team invitation modal has a url that you can copy and send to your team
- [ ] there is no way on God's green earth that you can copy anything but the entirety of the url
- [ ] visiting that link as someone who's already on the team takes you to the team page
- [ ] visiting that link as someone who's a member of parabol but not on the team takes you to the team page
- [ ] visiting that link as someone who isn't logged in shows you a generic authentication page that defaults to the "Create Account" view
- [ ] Can switch views to "Sign In" and invite token is still in the URL
- [ ] Can switch views to "Forgot password" and invite token is still in the URL
- [ ] Can submit a forgotten password, go back, then go back to sign in & still the invite token is in the URL
- [ ] can login & get redirected to the team page
- [ ] can sign up and get redirected to the team page
- [ ] the authentication page background looks like a meeting always (it's prettier) & when creating an account, it redirects you to the meeting
- [ ] if you change the token to something wrong like "foo" in the url, it tells you the token is invalid
- [ ] if the token is expired (massInviteToken.ts, change LIFESPAN to eg `ms('1s')`, refresh modal to get new token link) it tells you the token is expired

### Notes

**Comment by jordanh** [comment]
Opened #3005, a super low priority bug

File: src/server/graphql/rootQuery.ts [hunk 1]
```diff
@@ -4,17 +4,19 @@ import {getUserId} from 'server/utils/authorization'
...
 import getRethink from 'server/database/rethinkDriver'
+import massInvitation from 'server/graphql/queries/massInvitation'
 
 export default new GraphQLObjectType({
   name: 'Query',
   fields: () => ({
     viewer: {
       type: User,
-      resolve: (source, args, {authToken, dataLoader}) => {
+      resolve: (_source, _args, {authToken, dataLoader}) => {
```
Comment (jordanh) [inline src/server/graphql/rootQuery.ts]: Any reason you decided to rename these?

File: src/server/utils/massInviteToken.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import {fromEpochSeconds, toEpochSeconds} from 'server/utils/epochTime'
+import {sign, verify} from 'jsonwebtoken'
+import ms from 'ms'
+
+// 'eyJhbGciOiJIUzI1NiJ9'
```
Comment (jordanh) [inline src/server/utils/massInviteToken.ts]: Delete comment

**Comment by jordanh** [comment]
@mattkrick have a look at my code review (a minor q, really) – then merge when ready

File: src/server/utils/massInviteToken.ts (hunk 2 above)
Comment (jordanh) [reply src/server/utils/massInviteToken.ts]: Could you comment what this is for if you want to keep it?

**Comment by jordanh** [comment]
Change, then merge when ready

File: src/server/graphql/rootQuery.ts (hunk 1 above)
Comment (mattkrick) [reply src/server/graphql/rootQuery.ts]: ts protocol is to underscore unused args

**Pull request merged**